### DISCUSSION TITLE
How to change Port on Ansible Installation

### DISCUSSION THREAD

**Discussion by kammateo** [AzuraCast/AzuraCast#5508] [Q&A]
Hello guys,

Is it possible to change the port 80 and 443? I need those ports for a webpage. I used the Ansible Installation, also I use NginX. I intalled the Ansible version because back in the days the Docker was not working.

Thanks and have a wonderful day!

**Comment by ghost** [comment]
in the root directory for Azuarcast you'll find the .env file which is where you make edits to the port assignment
/var/azuracast/.env
